crates/api/src/lib.rs

```diff
@@ -1582,6 +1670,162 @@ mod test {
         }
     }
 
+    // Creates a mock file containing JSON-LD of the ChronicleOperations
+    // that would be created by the given command, although not in any particular order.
+    fn test_create_agent_operations_import() -> assert_fs::NamedTempFile {
+        let file = assert_fs::NamedTempFile::new("import.json").unwrap();
```

Comment:
It'd be lovely if we could specify this path on the command line. (Or, if not, I wonder if stdin would at least let us write `</my/things.json`.)